narayana/scripts/hudson/bm.sh

```diff
@@ -20,6 +23,8 @@ function cmp_narayana {
   fi
 
   echo -e "${1}\t${2}\t${3}\t${4}\t${tput}\t${gt}" >> results.txt
+  
+  return $res
```

Comment:
A nitpick `res` is a global variable and the return value from the function `runs` is never used so this statement is superfluous.

Reply:
You are right! Thanks @mmusgrov